### DISCUSSION TITLE
Scaffold DBContext - cannot choose Tables (or Schemas)

### DISCUSSION THREAD

**Discussion by LouisMilesRH** [JetBrains/rider-efcore#169] [Q&A]
Really like the Scaffold DBContext option however it doesn’t seem to pick up any Tables from my database which is odd… I added the NuGet package EntityFrameworkCore.SqlServer (which then appeared as a Provider) and added a connection string to appsettings.json (which then appeared in connection) and when I run the code it correctly generates all my tables. However when I go to the "Tables" tab within Rider and untick the "Scaffold all tables" option – it does not find any of these tables to toggle on and off (the same is true for Schemas).

![image](https://github.com/seclerp/rider-efcore/assets/59727334/9d050b48-8b62-4191-ab28-072fc6f02857)

![image](https://github.com/seclerp/rider-efcore/assets/59727334/2fd5a39b-a379-44ea-a944-7038e8b0ae56)

![image](https://github.com/seclerp/rider-efcore/assets/59727334/eadc4e78-5c03-4e48-94f4-0f3be0b40b08)

It is worth noting that I did have to add "TrustServerCertificate=True" to my connection string (unsure if this could be related) as it is a dev server and does not have an SSL cert on it.

Have I done something wrong or is this a bug?

**Comment by seclerp** [comment]
Hi @LouisMilesRH, this is currently not working as you expect. In the tables above you should manually enter your table and schema names according to the database.

But exploring the database to get out defined tables & schemas is a nice feature request, I will convert this to a suggestion, thanks!

**Comment by FerdinandoCavaliere** [comment]
I opened an issue because is not usefull add manually the tablet. I'd like a way as the extension "ef core power tools" in visual studio .
